types/k6/experimental/browser.d.ts

```diff
@@ -765,6 +765,53 @@ export interface BrowserContext {
     ): Page | null;
 }
 
+/**
+ * {@link ConsoleMessage} objects are dispatched by page via the
+ * `page.on('console')` event. For each console messages logged in the page
+ * there will be corresponding event in the k6 browser context.
```

Comment:
```suggestion
 * `page.on('console')` event. For each console message logged in the page
 * there will be a corresponding event in the k6 browser context.
```

Reply:
Tbh, i think this wording is a bit confusing. What is the "k6 browser context"?

Reply:
The intention is to explain that for each console event that happens in the context of the page in the browser instance, a representation of that event will be "passed" as input to the defined handler in the k6 browser test script context.

I'm also doubtful of how to describe the "k6 browser test script execution context". Any suggestion?

Reply:
> What about:
> "The k6 browser receives and delivers console messages logged on browser pages to the registered handlers."

What about a mix such as:
_"For each console message logged in the page, k6 browser delivers it to the registered handlers"._

Reply:
I like _"For each console message logged in the page, k6 browser delivers it to the registered handlers"_.

Reply:
It was a nice brainstorming :)

Reply:
Fixed in [a99c3cf](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/66549/commits/a99c3cfa84a548103a8aaaa78b10ec043130dc09).